src/components/interaction/interaction-contoller.ts

```diff
@@ -0,0 +1,36 @@
+import Provider from "oidc-provider";
+import { Request, Response } from "express";
+import { Config } from "../../config.js";
+
+export const interactionController =
+  (provider: Provider) => async (req: Request, res: Response) => {
+    const accountId = Config.getInstance().getSub();
+    const interactionDetails = await provider.interactionDetails(req, res);
+
+    if (interactionDetails.prompt.name === "login") {
+      //Fake Login
+      return provider.interactionFinished(req, res, {
+        login: { accountId },
+      });
+    } else if (interactionDetails.prompt.name === "consent") {
```

Comment:
Is this something we need to implement? We no longer do any form of consent in one login

Reply:
Have removed the prompt step altogether, hopefully aligns a bit more with what OL does